src/sandbox/cgroupFactory.go

```diff
@@ -186,6 +186,64 @@ func (cg *Cgroup) setFreezeState(state string) error {
 	}
 }
 
+// get mem usage in bytes
+func (cg *Cgroup) getMemUsage() (int64, error) {
```

Comment:
Would be nice to have the units (e.g., KB, MB) as part of the name of these functions